nf-test.config

```diff
@@ -3,14 +3,20 @@ config {
     testsDir "."
 
     // nf-test directory including temporary files for each test
-    workDir "/tmp"
-
-    // location of library folder that is added automatically to the classpath
-    libDir "tests/lib/"
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
 
     // location of an optional nextflow.config file specific for executing tests
-    configFile "nextflow.config"
+    configFile "tests/nextflow.config"
```

Comment:
what else do you have there that's not already in the profile test?

Reply:
thats how I have it - https://github.com/nf-core/methylseq/blob/0b8372017fad8e7f8cdc65a63baa326381ce8d60/nf-test.config#L9

how come you are seeing this diff ?